intake/__init__.py

```diff
@@ -19,6 +19,7 @@
 from .catalog import local
 from .catalog.default import load_combo_catalog
 from .catalog.local import MergedCatalog, EntrypointsCatalog
+from .catalog.zarr import ZarrCatalog
```

Comment:
Historically, all drivers core to Intake registered themselves at import and are accessible in the top-level. New drivers, though, usually in separate packages, can declare themselves with entrypoints in setup.py/toml, so they don't get imported until needed.

Reply:
OK, I saw some autoregistration code but wasn't sure. Happy to do what you recommend.